glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -585,6 +585,21 @@ open class GleanInternalAPI internal constructor () {
         }
     }
 
+    /**
+     * Set a tag to be applied to headers when uploading pings for debug view.
+     * This is only meant to be used internally by the `GleanDebugActivity`.
+     *
+     * @param value The value of the tag, which must be a valid HTTP header value.
+     */
+    fun setDebugViewTag(value: String): Boolean {
+        if (isInitialized()) {
```

Comment:
This might be intermittently failing when QA is testing, which is bad. Can we do as we do in [setUploadEnabled](https://searchfox.org/glean/rev/c6e1349fdb05baec59aaa5f465ba65e6cdd3b7d3/glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt#289) and cache the tag internally, then apply it when glean inits?

Reply:
Filed a bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1648009